core/src/data/key.rs

```diff
@@ -158,6 +160,11 @@ impl Key {
                     .copied()
                     .collect::<Vec<_>>()
             }
+            Key::U8(v) => [VALUE, 0]
```

Comment:
`U8` must be 1 because it is unsigned.
Additionally, the test code is missing. It would be nice if you could fill it out.